rotkehlchen/db/dbhandler.py

```diff
@@ -3455,13 +3459,11 @@ def get_blockchain_account_label(self, chain_address: ChainAddress) -> str | Non
         """Returns the label for a specific blockchain account.
         Returns None if either there is no label set or the account doesn't exist in database.
         """
-        query = 'SELECT label FROM blockchain_accounts WHERE account=? AND blockchain=?'
-        bindings = (chain_address.address, chain_address.blockchain.value)
-
         with self.conn.read_ctx() as cursor:
-            cursor.execute(query, bindings)
-            result = cursor.fetchone()
-
+            result = cursor.execute(
+                'SELECT name FROM address_book WHERE address=? AND blockchain=?',
+                (chain_address.address, chain_address.blockchain.value),
+            ).fetchone()
```

Comment:
This is now a query into the addressbook. Maybe would be better to move this method to `DBAddressbook` and have it like `query_single_chain_address_label`